config/crd/bases/multicluster.x-k8s.io_serviceexports.yaml

```diff
@@ -114,9 +114,3 @@ spec:
         type: object
     served: true
     storage: true
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
why did we remove this?

Reply:
`controller-gen` removes this between version 0.8 and 0.9.2. It's a bug that it was being added:
https://github.com/kubernetes-sigs/controller-tools/issues/456
https://github.com/kubernetes-sigs/controller-tools/pull/63